tests/components/homematicip_cloud/test_sensor.py

```diff
@@ -111,7 +112,7 @@
     )
 
     assert ha_state.state == "40"
-    assert ha_state.attributes["unit_of_measurement"] == "%"
+    assert ha_state.attributes["unit_of_measurement"] == UNIT_PERCENTAGE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5093397)